app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Table as NeetoUITable, Dropdown, Avatar } from "neetoui";
+
+import { DEFAULT_PAGE_INDEX, DEFAULT_PAGE_SIZE, CONTACTS } from "./constants";
+import { renderColumnData } from "./utils";
+
+const Table = ({ setIsDeleteAlertOpen, contacts = [] }) => {
+  const [pageNo, setPageNo] = useState(DEFAULT_PAGE_INDEX);
+  let columnData = renderColumnData();
+
+  columnData = [
+    {
+      title: "Name & Role",
+      dataIndex: "name",
+      key: "name",
+      render: name => (
+        <div className="flex flex-row items-center">
+          <Avatar user={{ name: name }} size="small" className="mr-2" />
+          {name}
+        </div>
+      ),
+    },
+    ...columnData.slice(1),
```

Comment:
Not sure why you are mutating the state. Rather wouldn't you be able to do `...renderColumnData().slice(1)` over line 25?